source/features/split-issue-pr-search-results.tsx

```diff
@@ -27,7 +27,10 @@ function init(): void {
 	cleanLinks();
 	pageSearchQuery = new SearchQuery(location);
 
-	const issueLink = select<HTMLAnchorElement>('nav.menu a[href*="&type=Issues"]')!;
+	const issueLink = select<HTMLAnchorElement>([
+		'nav.menu a[href*="&type=Issues"]', // Pre "Repository refresh" layout
```

Comment:
Is it? This page is not technically a repo page. I think this should just be updated

Reply:
Your right but enterprise needs it. I wanted to keep the same wording that when we drop the support for it we can drop em all.

Reply:
We can’t support 2 versions of everything. The repo refresh was an exception since it affected a lot of central features. GHE will just have to miss this minor feature for the time being.

Reply:
Since it’s so minor we *could* keep the double selector, but it needs to be marked as GHE, not the current comment.

Reply:
```suggestion
		'nav.menu a[href*="&type=Issues"]', // Todo remove when GHE selector is updated - (October 2020)
```